hooks/environment

```diff
@@ -15,8 +15,8 @@ source /etc/profile.d/modules.sh
 
 export SLURM_BUILDKITE_PATH="$(dirname "${BUILDKITE_BIN_PATH}")"
 export SHARED_DEPOT="${BUILDKITE_BUILD_PATH}/shared_depot"
-export GIT_SSH_COMMAND="ssh -i ${SLURM_BUILDKITE_PATH}/.ssh/id_rsa -o IdentitiesOnly=yes -o UserKnownHostsFile=${SLURM_BUILDKITE_PATH}/.ssh/known_hosts"
 export CI_BUILD_DIR="${BUILDKITE_BUILD_PATH}/${BUILDKITE_PIPELINE_SLUG}/${BUILDKITE_BUILD_NUMBER}"
+export JULIA_DEPOT_PATH="${JULIA_DEPOT_PATH:-$CI_BUILD_DIR/depot/default}:${SHARED_DEPOT}"
```

Comment:
We have to think about this, especially with relocatable depots coming with julia 1.11